### DISCUSSION TITLE
In Image Panel support H265 rosbag

### DISCUSSION THREAD

**Discussion by Jooolee** [foxglove/community#788] [Q&A]
Because, robot image data is very large, we use h265 compress.

we use play in studio, in Image Panel, but nothing, it is failure!!!

when we can use Image Panel play H265 rosbag?

![image](https://github.com/foxglove/community/assets/37429047/386071f7-cf65-43ac-bdf4-c4b2c69bfa1b)

**Comment by defunctzombie** [comment]
We do not support h265 playback in Studio. Documentation for what we support for the Image panel is here: https://foxglove.dev/docs/studio/panels/image

**Reply by amacneil** [reply]
Also - we're working on this and will hopefully support it in the near future

**Reply by Jooolee** [reply]
I am very expect it.

**Reply by jtbandes** [reply]
Correction – see also https://github.com/orgs/foxglove/discussions/794#discussioncomment-6605921 – "We are working on H264 support but will not have H265."
